org-docs/PROJECT-CHARTER.md

```diff
@@ -0,0 +1,136 @@
+# Abstract
+
+This Project is consensus-based and belongs in the *NiPreps (NeuroImaging PREProcessing toolS)* Community.
+Anyone with an interest in the Project can join the Community, contribute to the Project design, and participate in the decision making process. This document describes how that participation takes place, how to find consensus, and how deadlocks are resolved.
+
+# Governance Policy
+
+This document provides the governance policy for the Project.
+In general, all *NiPreps Projects* embrace a [liberal contribution model](https://opensource.guide/leadership-and-governance/#what-are-some-of-the-common-governance-structures-for-open-source-projects) of governance structure.
+However, because of the scientific domain of NiPreps , the community features some structure from *meritocracy models* to prescribe [the order in the authors list of new papers](https://www.nipreps.org/community/CONTRIBUTING/#publications) about these tools.
```

Comment:
```suggestion
However, because of the scientific domain of NiPreps, the community features some structure from *meritocracy models* to prescribe [the order in the authors list of new papers](https://www.nipreps.org/community/CONTRIBUTING/#publications) about these tools.
```